Library/Formula/joshua.rb

```diff
@@ -0,0 +1,62 @@
+class Joshua < Formula
+  desc "An open-source statistical machine translation decoder."
+  homepage "http://joshua-decoder.org/"
+  url "https://cs.jhu.edu/~post/files/joshua-6.0.5.tgz"
+  sha256 "e9aa6806017259ffe3e0c80b8f064cf2221b5618954e7fb7164117952bd6d265"
+  head "https://github.com/joshua-decoder/joshua.git"
+
+  option "with-language-packs", "Build with all available community language packs (es-->en and ar-->en)"
+  option "with-es-en-phrase-pack", "Build with Spanish–English phrase-based model [1.9 GB]."
+  option "with-ar-en-phrase-pack", "Build with Arabic–English phrase-based model [2.4 GB]."
+
+  depends_on :java
+  depends_on "ant" => :build
+  depends_on "boost" => :build
+  depends_on "md5sha1sum" => :build
+  depends_on :python => :build if MacOS.version <= :snow_leopard
+
+  resource "es-en-phrase-pack" do
+    url "http://cs.jhu.edu/~post/language-packs/language-pack-es-en-phrase-2015-03-06.tgz"
+    sha256 "213e05bbdcfbfa05b31e263c31f10a0315695fee26c2f37b0a78fb918bad9b5d"
+  end
+
+  resource "ar-en-phrase-pack" do
+    url "http://cs.jhu.edu/~post/language-packs/language-pack-ar-en-phrase-2015-03-18.tgz"
+    sha256 "2b6665b58b11e4c25d48191d3d5b62b7c591851a9767b14f9ccebf1951fddf90"
+  end
+
+  resource "indian-parallel-corpora" do
+    url "https://github.com/joshua-decoder/indian-parallel-corpora/archive/1.0.tar.gz"
+    sha256 "b6472648d9c13d72682554fd2434e2dc124433b6f6d9f39c16859759f30720c6"
+  end
+
+  def install
+    ENV["JOSHUA"] = "#{libexec}"
+    rm Dir["lib/*.{gr,.tar.gz"]
+    rm_rf "lib/README"
+    rm_rf "bin/.gitignore"
+    head do
+      system "ant"
+    end
+    if build.with? "language-packs"
+      resource("es-en-phrase-pack").stage { share.install "language-pack-es-en-phrase-2015-03-06", share/"joshualanguagepacks" }
+      resource("ar-en-phrase-pack").stage { share.install "language-pack-ar-en-phrase-2015-03-18", share/"joshualanguagepacks" }
+    elsif build.with? "es-en-phrase-pack"
+      resource("es-en-phrase-pack").stage { share.install "language-pack-es-en-phrase-2015-03-06", share/"joshualanguagepacks" }
+    elsif build.with? "ar-en-phrase-pack"
+      resource("ar-en-phrase-pack").stage { share.install "language-pack-ar-en-phrase-2015-03-18", share/"joshualanguagepacks" }
+    end
+    libexec.install Dir["*"]
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    resource("indian-parallel-corpora").stage { share.install "bn-en" }
+    system "$JOSHUA/bin/pipeline.pl --source bn --target en \
```

Comment:
I smartened this up to look this
```
    args = [
      "--source", "bn",
      "--target", "en",
      "--no-prepare",
      "--aligner", "berkeley",
      "--type", "hiero",
      "--corpus", "#{testpath}/bn-en/tok/training.bn-en",
      "--tune", "#{testpath}/bn-en/tok/dev.bn-en",
      "--test", "#{testpath}/bn-en/tok/devtest.bn-en"
    ]
    system "#{bin}/pipeline.pl", *args
```
i hope that is OK, I saw the formatting used pervasively throughout other Formulas.